### ISSUE TITLE
Nepal: Kathmandu valley area

### ISSUE THREAD

**Issue by smit1678** [hotosm/imagery-requests#8]
Kathmandu valley area identified as potential area for updated imagery. 

Candidates tagged. 

![](https://api.monosnap.com/rpc/file/download?id=6TG90ZLy1j9tyMn5Y2kYvPOKPi3Z1F)

ref #6

**Comment by camillacaros** [comment]
Two 2014 WV2 scenes covering the Kathmandu urban area were available for fast download from DG are now being processed. Will post progress asap.

![screen shot 2015-04-26 at 10 25 47 am](https://cloud.githubusercontent.com/assets/3768909/7338374/acea65a0-ebfe-11e4-918b-74c3fdf45b5a.png)

**Comment by smit1678** [comment]
It was identified that some pre-existing imagery is still available via the [task#190](http://tasks.hotosm.org/project/190). We'll change focus to look at post-disaster imagery for this area instead of pre-disaster. @camillacaros update here when you have a chance to look at post-disaster imagery.

**Comment by d3netxer** [comment]
FYI the imagery date for http://tasks.hotosm.org/project/190#task/166 is 30 Oct 2012 and it is WorldView 2. Are you guys proceeding with the two 2014 WV2 scenes? It is probably worth it to have newer pre-disaster imagery. btw, how are you planning on using post-disaster imagery?

**Comment by camillacaros** [comment]
@d3netxer we are not proceeding with teh 2014 images, an image captured Sunday (pretty cloudy) by DG is up and available per https://www.mapbox.com/blog/nepal-earthquake-imagery/  and now has improved color from what is seen in blog post. 

### next

we have acquired 3 scenes captured earlier today from DG, which are less cloud/hazy than yesterday's imagery. processing these now for upload. [coverage seen here](https://gist.github.com/anonymous/b8b425a894869dfc1e95), will add to Umap & will update as soon as these are processed & available for tracing.

**Comment by camillacaros** [comment]
Umap updated -- green region covering Kathmandu and a little ways east is what we are processing right now http://umap.openstreetmap.fr/en/map/hot-nepal-imagery-coord_37964#10/27.6081/85.2484

**Comment by d3netxer** [comment]
Today's WV3 shots are great; 

We need to be aware what we could be possibly duplicating with what the Pleiades imagery is showing for http://tasks.hotosm.org/project/1008

It is good your extent covers more area, especially to the east of what the Plaiades image is covering.

It also might be a good idea to follow-up with a WV3 shot 2-3 days from now, to capture how camps/temp structures are changing

**Comment by camillacaros** [comment]
@d3netxer yeah, we will have our eyes on new imagery coming in all week. unfortunately, the weather doesn't look great until ~Friday, but we'll see.  Thanks for the heads up on Pleiades imagery. wv3 images are off and running in processing now, so layer will be up soon either way at this point.

**Comment by camillacaros** [comment]
Imagery render of WV3 strips captured by DG earlier today looks finished -- mapid:  `mapbox.satellite-kathmandu-20150425-after` . will upate umap with tms url as well.

**Comment by smit1678** [comment]
Nice work @camillacaros. Just to document, these are WV3 27 April 2015 images. 

Putting tms link here as well: 

https://{switch:a,b,c,d}.tiles.mapbox.com/v4/mapbox.satellite-kathmandu-20150425-after/10//.jpg?access_token=pk.eyJ1Ijoib3BlbnN0cmVldG1hcCIsImEiOiJhNVlHd29ZIn0.ti6wATGDWOmCnCYen-Ip7Q

**Comment by maning** [comment]
The url above doesn't work for JOSM.

**Comment by maning** [comment]
Banepa section is used here: http://tasks.hotosm.org/project/1010

**Comment by d3netxer** [comment]
It is a cloudy day today around Kathmandu, I don't see any clear shots. We are starting to process newer before-disaster imagery around Kathmandu (Mainly Mar 2015 and Nov 2014). It could come in handy for follow-up IDP mapping using before/after imagery

**Comment by camillacaros** [comment]
Working on some more before & after imagery ... trying to see what is cloudfree and may be useful, will report soon
